BlueShift-iOS-SDK/InApps/BlueShiftNotificationSlideBannerViewController.m

```diff
@@ -178,13 +178,20 @@ - (void)initializeNotificationView {
             CGFloat messageBottomPadding = (messagePadding && messagePadding.bottom > 0) ? messagePadding.bottom : 0.0;
             CGFloat descriptionLabelHeight = descriptionLabel.frame.size.height + (messageTopPadding + messageBottomPadding);
             
-            if (descriptionLabelHeight < 50) {
-                descriptionLabelHeight = iconLabel.frame.size.height > 0 ? iconLabel.frame.size.height : imageView.frame.size.height;
-                //Modify height of description label if height is less than 50
-                //to align text veritically center
-                CGRect derscriptionLabelFrame = descriptionLabel.frame;
-                derscriptionLabelFrame.size.height = descriptionLabelHeight - messageTopPadding - messageBottomPadding;
-                descriptionLabel.frame = derscriptionLabelFrame;
+            // Set height to icon height/image height/default min height if label height is less than min height
+            // to align text veritically center
+            if (descriptionLabelHeight < kSlideInInAppNotificationMinimumHeight) {
+                if (iconLabel.frame.size.height > 0) {
+                    descriptionLabelHeight = iconLabel.frame.size.height;
+                } else if (imageView.frame.size.height > 0) {
+                    descriptionLabelHeight = imageView.frame.size.height;
+                } else {
+                    descriptionLabelHeight = kSlideInInAppNotificationMinimumHeight;
+                }
+                
+                CGRect descriptionLabelFrame = descriptionLabel.frame;
+                descriptionLabelFrame.size.height = descriptionLabelHeight - messageTopPadding - messageBottomPadding;
```

Comment:
ok, so the padding will be applied if provided. right?

Reply:
Yes, with top and bottom padding(line 179) if label height is below min-height(50) then it will be set back to icon/image/min-height with the given top bottom padding.